sycl/include/CL/sycl/ONEAPI/reduction.hpp

```diff
@@ -1659,7 +1642,30 @@ template <typename BinaryOperation, typename AccumulatorT>
 inline constexpr AccumulatorT known_identity_v =
     known_identity<BinaryOperation, AccumulatorT>::value;
 #endif
-
```

Comment:
I think it is worth to mark the code that we want to remove when allowed to break API.